api/admin/get_locale_index.js

```diff
@@ -0,0 +1,18 @@
+import template from '../../templates/server/locale';
+import config from '../../lib/config';
+import stylesheet from '../../lib/stylesheet';
+
+module.exports = () => ({
+  method: 'GET',
+  path: '/admin/locale',
+  config: {
+    auth: false
```

Comment:
Why is this one false, but below is 'jwt'?

Reply:
This route does not require authentication since it's a view, if an user is not authenticated the API call will fail and the SPA will redirect to the login.